apps/builder/app/canvas/shared/commands.ts

```diff
@@ -0,0 +1,6 @@
+import { createCommandsEmitter } from "~/shared/commands-emitter";
+
+export const { emitCommand, subscribeCommands } = createCommandsEmitter({
```

Comment:
emitCommand seems like has no typings support, I can emit anything
emitCommand("anything");

Reply:
I can't
<img width="828" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5635476/1c623f3a-7ece-42b4-9734-f374606d395d">

Reply:
<img width="663" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5077042/bd8e2653-6343-4417-9b86-c6da34d85528">